common/workunit/workunit.cpp

```diff
@@ -7460,11 +7652,23 @@ IUserDescriptor *CLocalWorkUnit::queryUserDescriptor() const
     CriticalBlock block(crit);
     if (!userDesc)
     {
-        SCMStringBuffer token, user, password;
-        getSecurityToken(token);
-        extractToken(token.str(), queryWuid(), user, password);
         userDesc.setown(createUserDescriptor());
-        userDesc->set(user.str(), password.str());
+        SCMStringBuffer token;
+        getDistributedAccessToken(token);
+        int rc = verifyWorkunitDAToken(token.str());
+        if (rc)
+        {
+            if (rc == 1)
+                ERRLOG("Workunit Distributed Access Token invalid");
+            else
+                ERRLOG("Workunit not active");
```

Comment:
If the developer doesn't know to change this, any new return codes will log "not active"?

Reply:
If there isn't a token, or the workunit isn't active does that really mean no userDesc should be created?  Or should the validation happen later?  And should the owner be the user?  If the owner is the user and they don't match I assume validation will fail when the token is used.

Reply:
Removed the call to verify the token, since that will happen later